openshift-storage-libs/openshiftstoragelibs/openshift_ops.py

```diff
@@ -423,13 +427,15 @@ def oc_create_app_dc_with_io(
                         the application PODs where constant I/O will run.
         dc_name_prefix (str): DC name will consist of this prefix and
                               random str.
+        cmd_args (list): list of arguments to be passed to the container
+                     command
         replicas (int): amount of application POD replicas.
         space_to_use (int): value in bytes which will be used for I/O.
         label (dict): dict of keys and values to add labels in DC.
         image (str): Container image for I/O.
     """
     return _oc_create_app_dc_with_io_image(
-        hostname, pvc_name, dc_name_prefix, replicas, space_to_use,
+        hostname, pvc_name, dc_name_prefix, cmd_args, replicas, space_to_use,
```

Comment:
`cmd_args` is optional parameter, need to pass correctly as `cmd_args=cmd_args`

Reply:
Done